app/views/planning_application/assessment_tasks/_check_consistency.html.erb

```diff
@@ -8,10 +8,12 @@
         Description, documents and proposal details
       </span>
     </li>
-    <li class="app-task-list__item">
-      <span class="app-task-list__task-name">
-        <%= link_to "History", planning_application_planning_history_path(@planning_application), class: "govuk-link" %>
-      </span>
-    </li>
+    <% if @planning_application.planning_history_enabled? %>
```

Comment:
You've made a change in the logic.

Have you added a test for this?

Reply:
I didn't add a spec as there was already a spec, what I did was passing true on the env [cc][1]

[1]: https://github.com/unboxed/bops/pull/761/files#diff-364c0ea3353cade165138baacbd0c41e0e4cea4ab8990073313e6eff7762ed2b